roommate/CalendarFetcher.fs

```diff
@@ -84,3 +85,31 @@ module CalendarFetcher =
             |> Seq.filter (fun (a,b,_) -> a.IsSome && b.IsSome)
             |> Seq.map (fun (a,b,c) -> a |> someOrBust |> hoursMinutes, b |> someOrBust |> hoursMinutes, c)
             |> Seq.iter (fun (a,b,c) -> printfn "  %s-%s  %s" a b c)
+            
+    let activateWebhook clientId clientSecret calendarId url =
+        let scopes = [CalendarService.Scope.CalendarReadonly]
+//        CalendarService.Scope.CalendarEvents
+        let tempFile = new FileDataStore("google-filedatastore", true)
+        
+        async {
+            let! credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
+                                ClientSecrets( ClientId = clientId, ClientSecret = clientSecret),
```

Comment:
This authentication mechanism is just for humans using apps, and it requires having an interactive session so the user can do the SSO workflow thing.

We'll need to use a different kind of authentication to connect to gcal from a lambda. I think it's a service account with an api key.